jetty-util/src/main/java/org/eclipse/jetty/util/BufferUtil.java

```diff
@@ -166,6 +167,23 @@ public static ByteBuffer copy(ByteBuffer buffer)
         return clone;
     }
 
+    /**
+     * Reset the buffer's endianness to {@link ByteOrder#BIG_ENDIAN}
+     * and clear the buffer to be empty in flush mode.
```

Comment:
Use 3rd person as per javadoc standards ("Resets", "clears"). Wrap with `<p>` the paragraphs.

Reply:
Ok.